src/js/script.js

```diff
@@ -0,0 +1,63 @@
+let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, ''];
+let indexOfVoid = 15;
+let winCombination1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+let winCombination2 = [1, 5, 9, 13, 2, 6, 10, 14, 3, 7, 11, 15, 4, 8, 12];
+
+function shuffle() {
+    let mySet = new Set();
+
+    function randomInteger(min, max) {
+        let rand = min + Math.random() * (max + 1 - min);
+        rand = Math.floor(rand);
+        return rand;
+    }
+
+    while (mySet.size < 15) {
+        mySet.add(randomInteger(1, 15));
+    }
+    arr = Array.from(mySet);
+    arr.push('');
+    fillGrid();
```

Comment:
Bad order of routines calls 
In my opinion routine that builds grid is more valuable and has more priority.
At that time shuffling routine is secondary. So 'fillGrid' should call 'shuffle' and use it`s result